src/core/Table/Table.tsx

```diff
@@ -218,6 +219,14 @@ export type TableProps<
    * @default 'default'
    */
   styleType?: 'default' | 'zebra-rows';
+  /**
+   * Virtualization is used for the scrollable table body.
+   * Height on the table is required for virtualization to work.
+   * @example
+   * <Table useVirtualization style={{height: 400}} {...} />
+   * @default false
+   */
+  useVirtualization?: boolean;
```

Comment:
I went through all the comments that were previously made on this PR and was surprised not to find any discussion about this prop name. It sounds like a hook. Maybe called it `virtualizeRows` or `enableVirtualization`?

It looks like MUI has a flipped boolean called `disableVirtualization` in their data grid, but I think that disables both column and row virtualization.

Reply:
renamed to `enableVirtualization`